ext/django_stubs_ext/db/models/manager.py

```diff
@@ -13,11 +13,12 @@
     from typing import Protocol, TypeVar
 
     _T = TypeVar("_T")
+    _M = TypeVar("_M")
```

Comment:
Let's call this `_Through` for consistency IMO

Reply:
Sure, I'll update